src/models/Polygon.js

```diff
@@ -0,0 +1,24 @@
+const { DataTypes } = require('sequelize');
+
+
+module.exports = (sequelize) => {
+    sequelize.define('Polygon', {
+        id: {
+            type: DataTypes.BIGINT(11),
+            primaryKey: true,
+            autoIncrement: true,
+            allowNull: false
+        },
+        points: {
+            type: DataTypes.STRING(2860),
+            defaultValue: null
+        },
+        type: {
+            type: DataTypes.BIGINT(11),
+            allowNull: false
+        }
+    },
+        {

```

Comment:
For a correct indent it should be four back space. Right?

Reply:
Can i improve the code and rebase it?